src/main/webapp/Portal.html

```diff
@@ -159,382 +159,70 @@
-              i.load(t)
-              return
-            }
-        }
-        function yt(n, t) {
-          var v, p, e
-          return n === r
-            ? (o ? f(t) : d.push(t), i)
-            : (s(n) && ((t = n), (n = 'ALL')), a(n))
-            ? ((v = {}),
-              u(n, function (n) {
-                v[n] = c[n]
-                i.ready(n, function () {
-                  y(v) && f(t)
-                })
-              }),
-              i)
-            : typeof n != 'string' || !s(t)
-            ? i
-            : ((p = c[n]), (p && p.state === l) || (n === 'ALL' && y() && o))
-            ? (f(t), i)
-            : ((e = h[n]), e ? e.push(t) : (e = h[n] = [t]), i)
-        }
-        function e() {
-          if (!r.body) {
-            n.clearTimeout(i.readyTimeout)
-            i.readyTimeout = n.setTimeout(e, 50)
-            return
-          }
-          o ||
-            ((o = !0),
-            vt(),
-            u(d, function (n) {
-              f(n)
-            }))
-        }
-        function k() {
-          r.addEventListener
-            ? (r.removeEventListener('DOMContentLoaded', k, !1), e())
-            : r.readyState === 'complete' &&
-              (r.detachEvent('onreadystatechange', k), e())
-        }
-        var r = n.document,
-          d = [],
-          h = {},
-          c = {},
-          ut =
-            'async' in r.createElement('script') ||
-            'MozAppearance' in r.documentElement.style ||
-            n.opera,
-          o,
-          g = (n.head_conf && n.head_conf.head) || 'head',
-          i = (n[g] =
-            n[g] ||
-            function () {
-              i.ready.apply(null, arguments)
-            }),
-          nt = 1,
-          ft = 2,
-          tt = 3,
-          l = 4,
-          p
-        if (r.readyState === 'complete') e()
-        else if (r.addEventListener)
-          r.addEventListener('DOMContentLoaded', k, !1),
-            n.addEventListener('load', e, !1)
-        else {
-          r.attachEvent('onreadystatechange', k)
-          n.attachEvent('onload', e)
-          p = !1
-          try {
-            p = !n.frameElement && r.documentElement
-          } catch (wt) {}
-          p &&
-            p.doScroll &&
-            (function pt() {
-              if (!o) {
-                try {
-                  p.doScroll('left')
-                } catch (t) {
-                  n.clearTimeout(i.readyTimeout)
-                  i.readyTimeout = n.setTimeout(pt, 50)
-                  return
-                }
-                e()
-              }
-            })()
-        }
-        i.load = i.js = ut ? lt : ct
-        i.test = ot
-        i.ready = yt
-        i.ready(r, function () {
-          y() &&
-            u(h.ALL, function (n) {
-              f(n)
-            })
-          i.feature && i.feature('domloaded', !0)
-        })
-      })(window)
-
-      let reactPath, reactDomPath, reactQueryPath, srcPath
```

Comment:
Remove react-query from this set of loaded scripts, since we'll import it another way (later)